templates/frontend-deployment.yaml

```diff
@@ -66,12 +66,21 @@ spec:
           tcpSocket:
             port: 8080
           timeoutSeconds: 1
+        volumeMounts:
+        - mountPath: "/tmp"
+          name: tmp
         resources:
           limits:
             cpu: {{ .Values.frontend.cpu | default "256m"}}
             memory: {{ .Values.frontend.memory | default "256Mi"}}
           requests:
             cpu: {{ .Values.frontend.cpu | default "256m"}}
             memory: {{ .Values.frontend.memory | default "256Mi"}}
+        securityContext:
+          {{ include "defaultSecurityContext" . | nindent 10 }}
       imagePullSecrets:
       - name: {{ .Release.Name }}-docker-cfg
+      volumes:
+      - name: tmp
+        emptyDir:
+          medium: Memory
```

Comment:
This `tmpfs` mount was needed for temp because Nginx is running on this node with:

```
    client_body_temp_path /tmp/client_temp;
    proxy_temp_path       /tmp/proxy_temp_path;
    fastcgi_temp_path     /tmp/fastcgi_temp;
    uwsgi_temp_path       /tmp/uwsgi_temp;
    scgi_temp_path        /tmp/scgi_temp;
```

I think it's only a concern if we had large file uploads routed to the front-end container, but I doubt that?

Reply:
We wouldn't expect any uploads to the frontend at all, so I don't see an issue here, unless maybe it opens a DoS vulnerability?

We have had one customer that can't use emptyDir mounts in their cluster for some reason, so we may run into an issue here. Although it's not clear what the rationale was or whether it was all emptyDir mounts or some specific configuration.